parsec/core/fs/userfs/userfs.py

```diff
@@ -756,7 +761,7 @@ async def _process_message(
             await self._process_message_sharing_revoked(msg)
 
         elif isinstance(msg, PingMessageContent):
-            self.event_bus.send("pinged", ping=msg.ping)
+            self.event_bus.send(CoreEvent.PINGED, ping=msg.ping)
```

Comment:
https://github.com/Scille/parsec-cloud/pull/1226#discussion_r446528076 ?

Reply:
renamed (i thought we were speaking about the event value string `"message.pinged"` and not the event name. )